packages/gatsby-theme-aio/src/components/Tabs/index.js

```diff
@@ -24,52 +27,379 @@ const animateIndicator = (indicator, animate) => {
   indicator.current.style.transition = animate ? '' : 'none';
 };
 
-const Tabs = forwardRef(({ children, className, onFontsReady, ...props }, ref) => {
+const setTabItemIcon = (child, className, iconSize) => {
+  if ( child?.props?.children?.props?.mdxType === "img") {
+    return <IconImage image={child} className={classNames(className, `spectrum-Icon--spectrum-icon-size-${iconSize}`)}/>;
+  }
+  return <Icons icon={child} className={classNames(className, `spectrum-Icon--spectrum-icon-size-${iconSize}`)}/>;
+};
+
+const setImageLoading = (child) => {
+  if (child?.props?.mdxType === "img") {
+    return cloneElement(child, {
+      loading: "eager"
+    });
+  }
+  return child;
+};
+
+const IconImage = ({ image = '', styles=''}) => {
+  return (image
+    ? cloneElement(image, {
+        children: cloneChildren(image.props.children, setImageLoading),
+        css: css`
+          display: flex;
+          align-items: center;
+          justify-content: center;
+          height: 100%;
+          width: 100%;
+          margin-top: 0;
+
+          img {
+            display: block;
+            object-fit: contain;
+          }
+          ${styles}
+        `,
+      })
+    : null);
+  }
+
+const Icons = ({ icon, styles }) => {
+  return(
+    icon
+      ? cloneElement(icon, {
+        css: css`
+          height: var(--spectrum-global-dimension-size-600);
+          width: var(--spectrum-global-dimension-size-600);
+          margin-top: 0;
+
+          img {
+            display: block;
+            object-fit: contain;
+          }
+          ${styles}
+        `
+      })
+      : null
+  )
+}
+
+const Tabs = forwardRef(({
+  orientation='horizontal',
+  density='regular',
+  isQuiet = true,
+  children,
+  className,
+  onFontsReady,
+  ...props },
+  ref) => {
+
   useEffect(() => {
     // Font affects positioning of the Tab indicator
     document.fonts.ready.then(() => {
       onFontsReady && onFontsReady();
     });
-  }, []);
+  }, [onFontsReady]);
 
   return (
     <div
       ref={ref}
       {...props}
-      className={classNames('spectrum-Tabs', 'spectrum-Tabs--horizontal', 'spectrum-Tabs--quiet', className)}
-      role="tablist">
+      className={classNames(className, 'spectrum-Tabs', `spectrum-Tabs--${orientation}`, { 'spectrum-Tabs--quiet' : isQuiet }, { 'spectrum-Tabs--compact': density === 'compact' })}
+    >
       {children}
     </div>
   );
 });
 
+const Item = forwardRef(({
+  elementType = 'div',
+  isDisabled=false,
+  isSelected=false,
+  className,
+  children,
+  icon,
+  label,
+  ...props }, ref
+) => {
+  const Element = elementType;
+  return (
+    <Element
+      {...props}
+      ref={ref}
+      role="tab"
+      aria-selected={isSelected ? true : false}
```

Comment:
`isSelected` is `bool` type so you can just write`isSelected` instead of checking the value.

Reply:
This is done